### PR TITLE
[suyabay] [#103078628] "Create project migraion design"

### PR DESCRIPTION
**Pull request by ibonly**
[suyabay] [#103078628] "Create project migraion design"

### CODE REVIEW

File: database/migrations/2014_10_12_000000_create_users_table.php [hunk 1]
```diff
@@ -29,6 +29,6 @@ public function up()
      */
     public function down()
     {
-        Schema::drop('users');
+        Schema::drop('channels');
```
Comment (unicodeveloper) [inline database/migrations/2014_10_12_000000_create_users_table.php:32]: Why did you change it from `users` to `channels` ? Is this not the Users table?

File: database/migrations/2014_10_12_000000_create_users_table.php [hunk 2]
```diff
@@ -13,8 +13,8 @@ class CreateUsersTable extends Migration
     public function up()
     {
         Schema::create('users', function (Blueprint $table) {
-            $table->increments('id');
-            $table->string('name');
+            $table->increments('user_id');
+            $table->string('username');
```
Comment (unicodeveloper) [inline database/migrations/2014_10_12_000000_create_users_table.php:17]: You should also make the `username` unique?

File: database/migrations/2015_09_14_135258_create_channels_table.php [hunk 3]
```diff
@@ -26,6 +28,6 @@ public function up()
      */
     public function down()
     {
-        Schema::drop('password_resets');
+        //
```
Comment (unicodeveloper) [inline database/migrations/2015_09_14_135258_create_channels_table.php:31]: This function is empty. Are you not meant to drop the `channels` table in this function?

File: database/migrations/2015_09_14_135400_create_views_table.php [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('views', function($table){
+            $table->increments('view_id');
+            $table->integer('guestUser_id')->unsigned();
```
Comment (unicodeveloper) [inline database/migrations/2015_09_14_135400_create_views_table.php:17]: Just change this to `user_id` .

File: database/migrations/2015_09_14_135439_create_comments_table.php [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::drop('comments');
+    }
```
Comment (unicodeveloper) [inline database/migrations/2015_09_14_135439_create_comments_table.php:34]: :+1:

File: database/migrations/2015_09_14_135422_create_favourites_table.php [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::drop('favourites');
+    }
```
Comment (unicodeveloper) [inline database/migrations/2015_09_14_135422_create_favourites_table.php:33]: Tight! Tight! Tight! :+1:

**Comment by unicodeveloper** [comment]
There is a grammatical blunder in the tile of your pull request and also in your commit message. It's `migration` instead of `migraion`

**Comment by unicodeveloper** [comment]
The commit message too doesn't follow the convention I specified. Check the @suyabay group. There is a particular format the commit message should follow.

**Comment by unicodeveloper** [comment]
Why is the name of your branch called `@ibraheem` ?. How does every team member know what feature you are working on?

**Comment by unicodeveloper** [comment]
I can't also see your avatar beside your commit. There is a default github avatar, that indicates that your git config is not set properly. Run `git config --list` via your terminal to check if your github email address and username is the required one for this account

File: database/migrations/2015_09_14_135258_create_channels_table.php (hunk 3 above)
Comment (ibonly) [inline database/migrations/2015_09_14_135258_create_channels_table.php:31]: I did not work on password_resets migration.

File: database/migrations/2014_10_12_000000_create_users_table.php (hunk 1 above)
Comment (ibonly) [inline database/migrations/2014_10_12_000000_create_users_table.php:32]: I was having issues with the migration files. It was later I realise that the migration is done according to the time the migration file was created. I saw the initial error but I thought I have corrected it.

File: database/migrations/2015_09_14_135258_create_channels_table.php (hunk 3 above)
Comment (unicodeveloper) [inline database/migrations/2015_09_14_135258_create_channels_table.php:31]: who worked on it?. I haven't accepted anyone's pull request yet, you're the first.
